crates/anvil-polkadot/src/api_server/server.rs

```diff
@@ -321,6 +325,14 @@ impl ApiServer {
                 node_info!("txpool_status");
                 self.txpool_status().await.to_rpc_result()
             }
+            EthRequest::TxPoolInspect(_) => {
```

Comment:
nit: Move the node_info in txpool_inspect to be similar to anvil. (This should be done for all methods at some point. I have added few myself x_X)